modules/ROOT/pages/building-https-proxy.adoc

```diff
@@ -0,0 +1,95 @@
+= Building an HTTPS API Proxy
+
+API Manager 2.x lets you use the secrets you stored in your secret group to build an HTTPS based API for your CloudHub or Hybrid deployments. +
+By configuring a TLS context for your inbound and outbound traffic, you can ensure your traffic's security.
+
+== Prerequisites
+
+To configure your HTTPS API Proxy using secrets stored in your secret group, you need to meet the following requirements:
+
+* Anypoint Security entitlements. +
```

Comment:
I pointed out this before in Slack also. this is not true. Please remove this

Reply:
This was removed.